mongodbatlas/data_source_mongodbatlas_projects.go

```diff
@@ -168,6 +200,11 @@ func flattenProjects(ctx context.Context, conn *matlas.Client, projects []*matla
 				fmt.Printf("[WARN] error getting project's api keys (%s): %s", project.ID, err)
 			}
 
+			limits, _, err := connV2.ProjectsApi.ListProjectLimits(ctx, project.ID).Execute()
+			if err != nil {
+				fmt.Printf("[WARN] error getting project's limits (%s): %s", project.ID, err)
```

Comment:
this seems like tech debt as it's the pattern in the file but this should probably be moved to a logger

Reply:
adjusted to generate warning diagnostics for these cases